### DISCUSSION TITLE
v1.5.0

### DISCUSSION THREAD

**Discussion by Triple-Z** [deb-sig/double-entry-generator#54] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/deb-sig/double-entry-generator/releases/tag/v1.5.0'>v1.5.0</a>.</em>

**Comment by klozzz** [comment]
小白求问mac如何配置环境运行translate...用go安装后在terminal/vs code都没有成功T T

**Reply by gaocegege** [reply]
可能需要配置一下环境变量，你是怎么安装的呀

**Reply by Triple-Z** [reply]
可以直接用 Release 的产物：https://github.com/deb-sig/double-entry-generator/releases/tag/v1.5.0

Mac 的话是这两个：
- 使用 ARM 的 CPU（如 M1）：[double-entry-generator_1.5.0_Darwin_arm64.tar.gz](https://github.com/deb-sig/double-entry-generator/releases/download/v1.5.0/double-entry-generator_1.5.0_Darwin_arm64.tar.gz)
- 使用 Intel 的 CPU（x86 架构）：[double-entry-generator_1.5.0_Darwin_x86_64.tar.gz](https://github.com/deb-sig/double-entry-generator/releases/download/v1.5.0/double-entry-generator_1.5.0_Darwin_x86_64.tar.gz)

下载下来直接用终端运行即可。

```bash
$ cd ~/Downloads
$ ./double-entry-generator version
```
